File: middle_letter/spec/middleletter_spec.rb
require 'middleletter'

describe '#middle_letter' do
  middleletter = MiddleLetter.new
  it 'returns the middle letters of an even word' do
    expect(middleletter.find_middle('test')).to eq('es')
  end

  it 'returns the middle letter of an odd word' do
    expect(middleletter.find_middle('tests')).to eq('s')
  end
end

File: middle_letter_javascript/spec/middleLetterSpec.js
describe('MiddleLetter', function(){
  var middleletter;

  beforeEach(function(){
      middleletter = new MiddleLetter()
  });

  describe('find_middle', function(){
    it('returns the middle letters of an even word', function(){
      expect(middleletter.find_middle('test')).toEqual('es');
    });

    it('returns the middle letter of an odd word', function(){
      expect(middleletter.find_middle('tests')).toEqual('s');
    })
  });
});
